<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DeviceManage</class>
 <widget class="QWidget" name="DeviceManage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>940</width>
    <height>528</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>30</y>
     <width>1317</width>
     <height>534</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="0,0,1,0">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_6">
      <property name="sizeConstraint">
       <enum>QLayout::SetFixedSize</enum>
      </property>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="bigSyslabel">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 75 20pt &quot;Adobe Arabic&quot;;
color: rgb(205, 0, 0);
background-color: rgb(205, 205, 193);</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer_5">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_2">
        <property name="styleSheet">
         <string notr="true">font: 75 12pt &quot;Courier New&quot;;</string>
        </property>
        <property name="title">
         <string>设备添加和删除</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_3" stretch="1,1">
           <item>
            <layout class="QVBoxLayout" name="verticalLayout">
             <property name="sizeConstraint">
              <enum>QLayout::SetFixedSize</enum>
             </property>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout">
               <item>
                <widget class="QLabel" name="label_2">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="font">
                  <font>
                   <family>Courier New</family>
                   <pointsize>12</pointsize>
                   <weight>9</weight>
                   <italic>false</italic>
                   <bold>false</bold>
                  </font>
                 </property>
                 <property name="text">
                  <string>系统名：</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QComboBox" name="cbxYear">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="minimumSize">
                  <size>
                   <width>200</width>
                   <height>0</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>200</width>
                   <height>16777215</height>
                  </size>
                 </property>
                 <item>
                  <property name="text">
                   <string>全部</string>
                  </property>
                 </item>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_2">
               <item>
                <widget class="QLabel" name="label_7">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="font">
                  <font>
                   <family>Courier New</family>
                   <pointsize>12</pointsize>
                   <weight>9</weight>
                   <italic>false</italic>
                   <bold>false</bold>
                  </font>
                 </property>
                 <property name="text">
                  <string>设备节点：</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QComboBox" name="cbxTerm">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="minimumSize">
                  <size>
                   <width>100</width>
                   <height>0</height>
                  </size>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_2">
             <property name="sizeConstraint">
              <enum>QLayout::SetFixedSize</enum>
             </property>
             <item>
              <widget class="QPushButton" name="btnAdd">
               <property name="font">
                <font>
                 <family>Courier New</family>
                 <pointsize>12</pointsize>
                 <weight>9</weight>
                 <italic>false</italic>
                 <bold>false</bold>
                </font>
               </property>
               <property name="text">
                <string>新 增</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="btnRemove">
               <property name="font">
                <font>
                 <family>Courier New</family>
                 <pointsize>12</pointsize>
                 <weight>9</weight>
                 <italic>false</italic>
                 <bold>false</bold>
                </font>
               </property>
               <property name="text">
                <string>移 除</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="btnSave">
               <property name="font">
                <font>
                 <family>Courier New</family>
                 <pointsize>12</pointsize>
                 <weight>9</weight>
                 <italic>false</italic>
                 <bold>false</bold>
                </font>
               </property>
               <property name="text">
                <string>保存修改</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="btnRedo">
               <property name="font">
                <font>
                 <family>Courier New</family>
                 <pointsize>12</pointsize>
                 <weight>9</weight>
                 <italic>false</italic>
                 <bold>false</bold>
                </font>
               </property>
               <property name="text">
                <string>撤销</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QGroupBox" name="groupBox">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="font">
                <font>
                 <family>Courier New</family>
                 <pointsize>12</pointsize>
                 <weight>9</weight>
                 <italic>false</italic>
                 <bold>false</bold>
                </font>
               </property>
               <property name="title">
                <string>信息</string>
               </property>
               <layout class="QVBoxLayout" name="verticalLayout_4">
                <property name="sizeConstraint">
                 <enum>QLayout::SetFixedSize</enum>
                </property>
                <item>
                 <layout class="QGridLayout" name="gridLayout">
                  <property name="rightMargin">
                   <number>10</number>
                  </property>
                  <item row="0" column="1">
                   <widget class="QLabel" name="SysNameLabel">
                    <property name="palette">
                     <palette>
                      <active>
                       <colorrole role="WindowText">
                        <brush brushstyle="SolidPattern">
                         <color alpha="255">
                          <red>0</red>
                          <green>85</green>
                          <blue>255</blue>
                         </color>
                        </brush>
                       </colorrole>
                      </active>
                      <inactive>
                       <colorrole role="WindowText">
                        <brush brushstyle="SolidPattern">
                         <color alpha="255">
                          <red>0</red>
                          <green>85</green>
                          <blue>255</blue>
                         </color>
                        </brush>
                       </colorrole>
                      </inactive>
                      <disabled>
                       <colorrole role="WindowText">
                        <brush brushstyle="SolidPattern">
                         <color alpha="255">
                          <red>120</red>
                          <green>120</green>
                          <blue>120</blue>
                         </color>
                        </brush>
                       </colorrole>
                      </disabled>
                     </palette>
                    </property>
                    <property name="font">
                     <font>
                      <family>Courier New</family>
                      <pointsize>12</pointsize>
                      <weight>9</weight>
                      <italic>false</italic>
                      <bold>false</bold>
                     </font>
                    </property>
                    <property name="text">
                     <string>0</string>
                    </property>
                   </widget>
                  </item>
                  <item row="1" column="0">
                   <widget class="QLabel" name="label_4">
                    <property name="text">
                     <string>设备节点总数：</string>
                    </property>
                   </widget>
                  </item>
                  <item row="2" column="1">
                   <widget class="QLabel" name="currentNodeLabel">
                    <property name="palette">
                     <palette>
                      <active>
                       <colorrole role="WindowText">
                        <brush brushstyle="SolidPattern">
                         <color alpha="255">
                          <red>198</red>
                          <green>0</green>
                          <blue>0</blue>
                         </color>
                        </brush>
                       </colorrole>
                      </active>
                      <inactive>
                       <colorrole role="WindowText">
                        <brush brushstyle="SolidPattern">
                         <color alpha="255">
                          <red>198</red>
                          <green>0</green>
                          <blue>0</blue>
                         </color>
                        </brush>
                       </colorrole>
                      </inactive>
                      <disabled>
                       <colorrole role="WindowText">
                        <brush brushstyle="SolidPattern">
                         <color alpha="255">
                          <red>120</red>
                          <green>120</green>
                          <blue>120</blue>
                         </color>
                        </brush>
                       </colorrole>
                      </disabled>
                     </palette>
                    </property>
                    <property name="font">
                     <font>
                      <family>Courier New</family>
                      <pointsize>12</pointsize>
                      <weight>9</weight>
                      <italic>false</italic>
                      <bold>false</bold>
                     </font>
                    </property>
                    <property name="text">
                     <string>0</string>
                    </property>
                   </widget>
                  </item>
                  <item row="2" column="0">
                   <widget class="QLabel" name="label_3">
                    <property name="text">
                     <string>当前节点：</string>
                    </property>
                   </widget>
                  </item>
                  <item row="1" column="1">
                   <widget class="QLabel" name="TotalNodeNumLabel">
                    <property name="palette">
                     <palette>
                      <active>
                       <colorrole role="WindowText">
                        <brush brushstyle="SolidPattern">
                         <color alpha="255">
                          <red>0</red>
                          <green>85</green>
                          <blue>255</blue>
                         </color>
                        </brush>
                       </colorrole>
                      </active>
                      <inactive>
                       <colorrole role="WindowText">
                        <brush brushstyle="SolidPattern">
                         <color alpha="255">
                          <red>0</red>
                          <green>85</green>
                          <blue>255</blue>
                         </color>
                        </brush>
                       </colorrole>
                      </inactive>
                      <disabled>
                       <colorrole role="WindowText">
                        <brush brushstyle="SolidPattern">
                         <color alpha="255">
                          <red>120</red>
                          <green>120</green>
                          <blue>120</blue>
                         </color>
                        </brush>
                       </colorrole>
                      </disabled>
                     </palette>
                    </property>
                    <property name="font">
                     <font>
                      <family>Courier New</family>
                      <pointsize>12</pointsize>
                      <weight>9</weight>
                      <italic>false</italic>
                      <bold>false</bold>
                     </font>
                    </property>
                    <property name="text">
                     <string>0</string>
                    </property>
                   </widget>
                  </item>
                  <item row="0" column="0">
                   <widget class="QLabel" name="label_6">
                    <property name="text">
                     <string>系统名：</string>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
               </layout>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Expanding</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_7">
      <property name="sizeConstraint">
       <enum>QLayout::SetFixedSize</enum>
      </property>
      <item>
       <spacer name="verticalSpacer_3">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="TableView" name="tableView">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>960</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>960</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Courier New</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(170, 170, 255);</string>
        </property>
        <property name="sizeAdjustPolicy">
         <enum>QAbstractScrollArea::AdjustIgnored</enum>
        </property>
        <property name="textElideMode">
         <enum>Qt::ElideMiddle</enum>
        </property>
        <property name="showGrid">
         <bool>false</bool>
        </property>
        <property name="gridStyle">
         <enum>Qt::NoPen</enum>
        </property>
        <property name="sortingEnabled">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer_4">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <spacer name="horizontalSpacer_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Maximum</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TableView</class>
   <extends>QTableView</extends>
   <header>tableview.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
